AntWars/Board/BoardObjects.cs

```diff
@@ -195,7 +195,8 @@ public bool move(BoardObject obj, Coordinates coords)
                 return false;
             }
             removeFromMap(obj);
-            obj.Coords = coords;
+            obj.Coords.X = coords.X;
+            obj.Coords.Y = coords.Y;
             addToMap(obj);
```

Comment:
Warum wurde hier in die Koordinatenebene gesplittet?
Objekt kann doch übergeben werden so wie es voher war.